CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Changes are grouped as follows
 - `Fixed` for any bug fixes.
 - `Security` in case of vulnerabilities.
 
+## Unreleased
```

Comment:
Good QOL addition! Let's bump patch version for this 👍

Reply:
I updated the CHANGELOG and bumped the version in the `pyproject.toml` and `_version.py`